OpenRA.Mods.Common/Activities/Air/Fly.cs

```diff
@@ -50,6 +59,22 @@ public Fly(Actor self, Target t, WPos? initialTargetPosition = null, Color? targ
 				lastVisibleTarget = Target.FromPos(target.CenterPosition);
 			else if (initialTargetPosition.HasValue)
 				lastVisibleTarget = Target.FromPos(initialTargetPosition.Value);
+
+			bodyOrientation = self.TraitOrDefault<BodyOrientation>();
+			facing = self.TraitOrDefault<IFacing>();
+			posture = self.TraitOrDefault<IPosture>();
+			currentFacing = lastFacing;
+			firstCall = true;
+			rollStep = WAngle.FromDegrees(5);
+			if (facing != null)
+			{
+				lastFacing = facing.Facing;
+			}
```

Comment:
https://github.com/OpenRA/OpenRA/wiki/Coding-Standard says remove the one liner braces.

Reply:
ok

Reply:
I will fix it in my reposite directly later.